Binary files liquid-dsp.orig/.git/index and liquid-dsp/.git/index differ
diff -urNp liquid-dsp.orig/configure.ac liquid-dsp/configure.ac
--- liquid-dsp.orig/configure.ac	2020-04-07 19:47:16.999978525 -0500
+++ liquid-dsp/configure.ac	2020-04-07 20:13:23.948011753 -0500
@@ -191,7 +191,7 @@ else
                        src/dotprod/src/dotprod_rrrf.av.o \
                        src/dotprod/src/dotprod_crcf.av.o \
                        src/dotprod/src/sumsq.o"
-        ARCH_OPTION="-fno-common -faltivec";;
+        ARCH_OPTION="-fno-common -maltivec";;
     armv1*|armv2*|armv3*|armv4*|armv5*|armv6*)
         # assume neon instructions are NOT available
         MLIBS_DOTPROD="src/dotprod/src/dotprod_cccf.o \
